src/Console/ConsoleInputArgument.php

```diff
@@ -89,6 +97,15 @@ public function __construct(
             $this->_required = $required;
             $this->_choices = $choices;
             $this->_default = $default;
+            $this->_separator = $separator;
+        }
+
+        // Separator could only be one character, not space.
+        if (isset($this->_separator)) {
+            if (mb_strlen($this->_separator) > 1) {
+                $this->_separator = mb_substr($this->_separator, 0, 1);
```

Comment:
I silent change doesn't sound too nice to me. Wouldn't this be a developer error and as such exception?

Reply:
Also, we could remove the comment and just assert on whitspace.
Then it can even separators of multiple chars where needed (e.g. including escaping sequence):

    if (str_contains($this->_separator, ' ')) { throw new ...Exception; }

Reply:
Fix both case